### PR TITLE
Fixes prepare_release.sh.

### PR DESCRIPTION
**Pull request by coollog**
Updates from fix at https://github.com/GoogleContainerTools/jib/pull/1129#issuecomment-429018740

### CODE REVIEW

File: scripts/prepare_release.sh [hunk 1]
```diff
@@ -4,46 +4,51 @@
...
-    local nextMinorVersion=$((minorVersion+1))
-    echo $version | sed "s/\([0-9][0-9]*\.[0-9][0-9]*\)\.[0-9][0-9]*/\1.$nextMinorVersion/"
+  local version=$1
+  local minorVersion=$(echo $version | sed 's/[0-9][0-9]*\.[0-9][0-9]*\.\([0-9][0-9]\)*/\1/')
+  local nextMinorVersion=$((minorVersion+1))
+  echo $version | sed "s/\([0-9][0-9]*\.[0-9][0-9]*\)\.[0-9][0-9]*/\1.$nextMinorVersion/"
 }
 
-[ $# -ne 2 ] || DieUsage
+[ $# -ne 1 ] && DieUsage
```
Comment (chanseokoh) [inline scripts/prepare_release.sh]: I see `$2` being used below. Should this be `[ $# -ne 1 ] && [ $# -ne 2 ] && DieUsage`?

Comment (coollog) [reply scripts/prepare_release.sh]: Ah yes, mis-copied this

**Comment by coollog** [comment]
Yes, for a common location to store our team's open source project infrastructure code (like the release script and probably the Kokoro scripts as well), we could either use:

- a separate repository on GitHub, or
- a separate internal repository

**Comment by briandealwis** [comment]
`https://cloud.google.com/_static/images/cloud/products/logos/svg/dlp.svg` was having a 503 moment.

**Pull request merged**